### PR TITLE
[FR] Add "Badge-weighted seeding"

### PR DESCRIPTION
**Pull request by Pisapou**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Tournaments/Badge-weighted_seeding/fr.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  - rank restricted
+  - rank restriction
+  - rank-restricted
+  - tournament seed
+  - tournament seeding
+---
+
+# Badge-weighted seeding
+
+**Badge-weighted seeding** (***BWS***) est un [seeding](https://en.wikipedia.org/wiki/Seed_(sports)) de [tournois](/wiki/Tournaments) et un système de restrictions qui prend en compte tous les [Badges de Profil](/wiki/Community/Profile_badge) de chaque joueur pour qu'ils comptent autant que leur [rang global](/wiki/Ranking#performance-points-ranking). Conçu à l'origine par ![][flag_US] [DigitalHypno](https://osu.ppy.sh/users/4384207), L'objectif de ce système était de "répartir plus précisément les joueurs et d'appliquer les restrictions de rang en fonction des badges de tournoi des joueurs",[^hippo-cup-bws] par rapport à la méthode de classement plus standard qui utilise directement le classement global des joueurs.
```
Comment (Pisapou) [inline wiki/Tournaments/Badge-weighted_seeding/fr.md:14]:
```suggestion
Le **Badge-weighted seeding** (***BWS***) est un [seeding](https://en.wikipedia.org/wiki/Seed_(sports)) de [tournois](/wiki/Tournaments) et un système de restrictions qui prend en compte tous les [Badges de Profil](/wiki/Community/Profile_badge) de chaque joueur pour qu'ils comptent autant que leur [rang global](/wiki/Ranking#performance-points-ranking). Conçu à l'origine par ![][flag_US] [DigitalHypno](https://osu.ppy.sh/users/4384207), l'objectif de ce système était de "répartir plus précisément les joueurs et d'appliquer les restrictions de rang en fonction des badges de tournoi des joueurs",[^hippo-cup-bws] par rapport à la méthode de classement plus standard qui utilise directement le classement global des joueurs.

```

File: wiki/Tournaments/Badge-weighted_seeding/fr.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Le principe de la BWS est que les joueurs se voient attribuer un meilleur classement s'ils ont gagné des badges pour avoir participé à des tournois dans le passé. De cette façon, les joueurs ne resteront pas indéfiniment au même rang s'ils sont performants en tournois, par rapport à d'autres joueurs de même rang.[^digitalhypno-discord-1][^digitalhypno-discord-2] Dans un ensemble de tournois qui utilisent tous la BWS, on s'attend à ce que les joueurs qui commencent à des rangs similaires tendent vers des têtes de série BWS qui reflètent mieux leur probabilité de gagner des matchs contre des joueurs de même rang.[^digitalhypno-discord-3][^digitalhypno-discord-4]
```
Comment (Pisapou) [inline wiki/Tournaments/Badge-weighted_seeding/fr.md:16]:
```suggestion
Le principe du BWS est que les joueurs se voient attribuer un meilleur classement s'ils ont gagné des badges pour avoir participé à des tournois dans le passé. De cette façon, les joueurs ne resteront pas indéfiniment au même rang s'ils sont performants en tournois, par rapport à d'autres joueurs de même rang.[^digitalhypno-discord-1][^digitalhypno-discord-2] Dans un ensemble de tournois qui utilisent tous le BWS, on s'attend à ce que les joueurs qui commencent à des rangs similaires tendent vers des têtes de série BWS qui reflètent mieux leur probabilité de gagner des matchs contre des joueurs de même rang.[^digitalhypno-discord-3][^digitalhypno-discord-4]

```

File: wiki/Tournaments/Badge-weighted_seeding/fr.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Le BWS a été utilisé pour la première fois en fin 2018 pour la [Hippo Cup 2](https://osu.ppy.sh/community/forums/topics/848153). Depuis, cela a été utilisé de nombreuses fois dans des [tournois communautaires](/wiki/Tournaments#community), et constitue l'une des principales méthodes de classement actuellement utilisées, avec les classements globaux et les tours de qualification.
+
+## Technique
+
+La méthode originale et la plus courante pour calculer les seeds des joueurs dans la BWS est la fonction de décroissance bornée suivante :
```
Comment (Pisapou) [inline wiki/Tournaments/Badge-weighted_seeding/fr.md:22]:
```suggestion
La méthode originale et la plus courante pour calculer les seeds des joueurs avec le BWS est la fonction de décroissance bornée suivante :

```

File: wiki/Tournaments/Badge-weighted_seeding/fr.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+```
+seed = global_rank ^ (0.9937 ^ (badge_count ^ 2))
+```
+
+- `badge_count` (≥ 0): Nombre de badges sur le profil du joueur qui ont été gagnés en tant que prix pour des tournois précédents du même [Modes de jeu](/wiki/Game_mode)
```
Comment (Pisapou) [inline wiki/Tournaments/Badge-weighted_seeding/fr.md:28]:
```suggestion
- `badge_count` (≥ 0): Nombre de badges sur le profil du joueur qui ont été gagnés en tant que prix pour des tournois précédents du même [modes de jeu](/wiki/Game_mode)

```

File: wiki/Tournaments/Badge-weighted_seeding/fr.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+- `global_rank` (≥ 1): [Classement des points de performance](/wiki/Ranking#performance-points-ranking) du joueur sur le classement global correspondant au [Modes de jeu](/wiki/Game_mode) du tournois.
```
Comment (Pisapou) [inline wiki/Tournaments/Badge-weighted_seeding/fr.md:29]:
```suggestion
- `global_rank` (≥ 1): [Classement des points de performance](/wiki/Ranking#performance-points-ranking) du joueur sur le classement global correspondant au même [modes de jeu](/wiki/Game_mode) du tournois.

```

File: wiki/Tournaments/Badge-weighted_seeding/fr.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+Cette fonction satisfait des propriétés importantes pour le BWS :
+
+- La seed d'un joueur diminue par incréments plus importants au fur et à mesure qu'il obtient des badges, ou en d'autres termes, chaque badge a plus de valeur que le précédent.[^hippo-cup-bws] Notez que dans la formule ci-dessus, ceci n'est vrai que jusqu'à environ 2 à 6 badges (selon le `global_rank`), où il y a un point d'inflexion, et alors l'effet devient opposé.
+- Les seed du BWS sont toujours inférieures ou égales aux `global_rank`. Ils sont égaux si le joueur n'a pas de badges.
+- Les seeds du BWS ont la même portée que le `global_rank`.
+
+Certains tournois varient leur mise en œuvre de la BWS en utilisant une fonction différente,[^brtt-bws] mais toutes partagent les mêmes propriétés décrites ci-dessus.
```
Comment (Pisapou) [inline wiki/Tournaments/Badge-weighted_seeding/fr.md:37]:
```suggestion
Certains tournois varient leur mise en œuvre du BWS en utilisant une fonction différente,[^brtt-bws] mais toutes partagent les mêmes propriétés décrites ci-dessus.

```

File: wiki/Tournaments/Badge-weighted_seeding/fr.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+## Inconvénients potentiels
+
+
+- Les BWS ne peuvent être mises à jour que lorsque les joueurs reçoivent des badges, et les badges ne sont généralement attribués qu'en tant que prix de première place à la fin des tournois. Ainsi, le processus d'établissement des joueurs dans des seeds leur correspondant peut être très lent.
```
Comment (Pisapou) [inline wiki/Tournaments/Badge-weighted_seeding/fr.md:42]:
```suggestion
- Les BWS ne peuvent être mises à jour que lorsque les joueurs reçoivent des badges, et les badges ne sont généralement attribués qu'en tant que prix de première place à la fin des tournois. Ainsi, le processus d'établissement des joueurs dans des seeds réprentatives de leur niveau peut être très lent.

```

**Comment by cl8n** [comment]
please leave the second task unchecked until another french speaker reviews the PR

File: wiki/Tournaments/Badge-weighted_seeding/fr.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
+---
+tags:
+  - badge weighting system
+  - BWS
+  - rank restricted
+  - rank restriction
+  - rank-restricted
+  - tournament seed
+  - tournament seeding
```
Comment (Biokax) [inline wiki/Tournaments/Badge-weighted_seeding/fr.md:9]:
```suggestion
  - tournament seeding
  - système de pondération des badges
  - restriction de rang
  - rang restreint
  - seed de tournoi
  - seeding de tournoi
```

**Review by Biokax — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/Badge-weighted_seeding/fr.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+  - système de pondération des badges
+  - restriction de rang
+  - rang restreint
+  - seed de tournoi
+  - seeding de tournoi
+---
+
+# Badge-weighted seeding
+
+Le **Badge-weighted seeding** (***BWS***) est un [seeding](https://en.wikipedia.org/wiki/Seed_(sports)) de [tournois](/wiki/Tournaments) et un système de restrictions qui prend en compte tous les [Badges de Profil](/wiki/Community/Profile_badge) de chaque joueur pour qu'ils comptent autant que leur [rang global](/wiki/Ranking#performance-points-ranking). Conçu à l'origine par ![][flag_US] [DigitalHypno](https://osu.ppy.sh/users/4384207), l'objectif de ce système était de "répartir plus précisément les joueurs et d'appliquer les restrictions de rang en fonction des badges de tournoi des joueurs",[^hippo-cup-bws] par rapport à la méthode de classement plus standard qui utilise directement le classement global des joueurs.
```
Comment (Y0hananas) [inline wiki/Tournaments/Badge-weighted_seeding/fr.md:19]:
```suggestion
Le **Badge-weighted seeding** (***BWS***) est un [seeding](https://en.wikipedia.org/wiki/Seed_(sports)) de [tournois](/wiki/Tournaments) et un système de restrictions qui prend en compte tous les [Badges de Profil](/wiki/Community/Profile_badge) de chaque joueur pour qu'ils comptent autant que leur [rang global](/wiki/Ranking#performance-points-ranking). Conçu à l'origine par ![][flag_US] [DigitalHypno](https://osu.ppy.sh/users/4384207), l'objectif de ce système était de "répartir plus précisément les joueurs et d'appliquer les restrictions de rang en fonction de leurs badges de tournoi",[^hippo-cup-bws] par rapport à la méthode de classement plus standard qui utilise directement leur classement global.

```

File: wiki/Tournaments/Badge-weighted_seeding/fr.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+
+Le principe du BWS est que les joueurs se voient attribuer un meilleur classement s'ils ont gagné des badges pour avoir participé à des tournois dans le passé. De cette façon, les joueurs ne resteront pas indéfiniment au même rang s'ils sont performants en tournois, par rapport à d'autres joueurs de même rang.[^digitalhypno-discord-1][^digitalhypno-discord-2] Dans un ensemble de tournois qui utilisent tous le BWS, on s'attend à ce que les joueurs qui commencent à des rangs similaires tendent vers des têtes de série BWS qui reflètent mieux leur probabilité de gagner des matchs contre des joueurs de même rang.[^digitalhypno-discord-3][^digitalhypno-discord-4]
```
Comment (Y0hananas) [inline wiki/Tournaments/Badge-weighted_seeding/fr.md:22]:
```suggestion
Le principe du BWS est que les joueurs se voient attribuer un meilleur classement s'ils ont gagné des badges pour avoir participé à des tournois dans le passé. De cette façon, les joueurs ne resteront pas indéfiniment au même rang s'ils sont performants en tournois par rapport à d'autres joueurs de même rang.[^digitalhypno-discord-1][^digitalhypno-discord-2] Dans un ensemble de tournois qui utilisent tous le BWS, on s'attend à ce que les joueurs qui commencent à des rangs similaires tendent vers des têtes de série BWS qui reflètent mieux leur probabilité de gagner des matchs contre des joueurs de même rang.[^digitalhypno-discord-3][^digitalhypno-discord-4]

```

File: wiki/Tournaments/Badge-weighted_seeding/fr.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+La méthode originale et la plus courante pour calculer les seeds des joueurs avec le BWS est la fonction de décroissance bornée suivante :
+
+
+```
+seed = global_rank ^ (0.9937 ^ (badge_count ^ 2))
+```
+
+- `badge_count` (≥ 0): Nombre de badges sur le profil du joueur qui ont été gagnés en tant que prix pour des tournois précédents du même [modes de jeu](/wiki/Game_mode)
+
+- `global_rank` (≥ 1): [Classement des points de performance](/wiki/Ranking#performance-points-ranking) du joueur sur le classement global correspondant au même [modes de jeu](/wiki/Game_mode) du tournois.
```
Comment (Y0hananas) [inline wiki/Tournaments/Badge-weighted_seeding/fr.md:38]:
```suggestion
- `global_rank` (≥ 1): [Classement des points de performance](/wiki/Ranking#performance-points-ranking) du joueur sur le classement global correspondant au même [modes de jeu](/wiki/Game_mode) du tournoi.

```

File: wiki/Tournaments/Badge-weighted_seeding/fr.md [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+
+
+Cette fonction satisfait des propriétés importantes pour le BWS :
+
+- La seed d'un joueur diminue par incréments plus importants au fur et à mesure qu'il obtient des badges, ou en d'autres termes, chaque badge a plus de valeur que le précédent.[^hippo-cup-bws] Notez que dans la formule ci-dessus, ceci n'est vrai que jusqu'à environ 2 à 6 badges (selon le `global_rank`), où il y a un point d'inflexion, et alors l'effet devient opposé.
+- Les seed du BWS sont toujours inférieures ou égales aux `global_rank`. Ils sont égaux si le joueur n'a pas de badges.
```
Comment (Y0hananas) [inline wiki/Tournaments/Badge-weighted_seeding/fr.md:44]:
```suggestion
- Les seeds du BWS sont toujours inférieures ou égales aux `global_rank`. Ils sont égaux si le joueur n'a pas de badges.

```

File: wiki/Tournaments/Badge-weighted_seeding/fr.md [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+- Les seeds du BWS ont la même portée que le `global_rank`.
+
+Certains tournois varient leur mise en œuvre du BWS en utilisant une fonction différente,[^brtt-bws] mais toutes partagent les mêmes propriétés décrites ci-dessus.
+
+
+## Inconvénients potentiels
+
+
+- Les BWS ne peuvent être mises à jour que lorsque les joueurs reçoivent des badges, et les badges ne sont généralement attribués qu'en tant que prix de première place à la fin des tournois. Ainsi, le processus d'établissement des joueurs dans des seeds réprentatives de leur niveau peut être très lent.
```
Comment (Y0hananas) [inline wiki/Tournaments/Badge-weighted_seeding/fr.md:53]:
```suggestion
- Les BWS ne peuvent être mises à jour que lorsque les joueurs reçoivent des badges, et les badges ne sont généralement attribués qu'en tant que prix de première place à la fin des tournois. Ainsi, le processus d'établissement des joueurs dans des seeds représentatives de leur niveau peut être très lent.

```

**Review by cl8n — APPROVED** [review]

**Pull request merged**